components/HomeButton.js

```diff
@@ -1,59 +1,64 @@
 import React from 'react';
 import styled from 'styled-components/native';
-import { View, Image, StyleSheet, TouchableOpacity, Text } from 'react-native';
 
+import { View, Image, StyleSheet, TouchableOpacity, Text } from 'react-native';
 const styles = StyleSheet.create({
     styledImage: {
-        width: 169,
-        height: 207,
+        minWidth: 165,
+        maxWidth: '30%',
+        height: 185,
         margin: 5,
-        top: 30,
         borderRadius: 15,
-        zIndex: 0
+        zIndex: 0,
+        top: 30
+    },
+    styledText: {
+        zIndex: 1,
+        position: 'absolute',
+        fontWeight: 'bold',
+        display: 'flex',
+        textAlign: 'center',
+        fontSize: 30,
+        minWidth: 165,
+        minHeight: 220,
+        padding: 10,
+        left: 5,
+        top: 70,
+        color: 'white',
+        position: 'absolute'
+    },
+    styledTextHow: {
+        zIndex: 1,
+        position: 'absolute',
+        fontWeight: 'bold',
+        display: 'flex',
+        textAlign: 'center',
+        fontSize: 30,
+        minWidth: 165,
+        minHeight: 220,
+        padding: 10,
+        left: 5,
+        top: 70,
+        color: '#EE3282',
+        position: 'absolute'
```

Comment:
Since these two are all the same except the color, you should probably create only one css where it takes color as a param. Don't have to change it in this PR. It would just be cleaner when u do it in the future to not repeat an entire css.

Reply:
Good point @sukritsangvong, thanks! I'll include these changes in the next update!